pyogrio/tests/conftest.py

```diff
@@ -16,7 +21,6 @@
     ".geojsonl": "GeoJSONSeq",
     ".geojsons": "GeoJSONSeq",
     ".gpkg": "GPKG",
-    ".json": "GeoJSON",
```

Comment:
Was this done just to reduce the test parametrization size?

Reply:
No, this was because newer versions of GDAL introduced additional drivers that use the `.json` extension, so we get an exception using it because there are multiple drivers available for it (which is what we want).  Seemed easier just to avoid that issue rather than adding version-specific handling.